src/System.Web.OData/OData/Routing/DefaultODataPathHandler.cs

```diff
@@ -112,12 +112,10 @@ private ODataPath Parse(string serviceRoot, string odataPath, IServiceProvider r
             {
                 Contract.Assert(serviceRoot != null);
 
-                serviceRootUri = new Uri(
-                    serviceRoot.EndsWith("/", StringComparison.Ordinal)
-                        ? serviceRoot
-                        : serviceRoot + "/");
+                string serviceRootStr = serviceRoot.EndsWith("/", StringComparison.Ordinal) ? serviceRoot : serviceRoot + "/";
+                serviceRootUri = new Uri(serviceRootStr);
```

Comment:
serviceRootUri is now unused, please delete it here and the definition above.

Reply:
Ah, missed it. My browser search was "serviceRootUri " instead of "serviceRootUri". Thanks.